web/public/templates/dojos/events/user-events.dust

```diff
@@ -3,42 +3,45 @@
   <div ng-if="isTicketingAdmin" class="alert bg-warning">
     <h5>{@i18n key="See"/} <a ui-sref="my-dojos"><b>{@i18n key="My Dojos"/}</b></a> {@i18n key="to create an event"/}</h5>
   </div>
-  <div class="row">
-    <p ng-if="!dojosEvents || dojosEvents.length === 0 || !currentEvents">{@i18n key="None of your Dojo's have any Events planned currently" /}</p>
-    <div class="my-dojos" ng-repeat="dojoEvents in dojosEvents | filter:hasEvents">
-      <legend ng-bind-html="dojoEvents.title"></legend>
-      <div class="row">
-        <div class="col-md-12 table-responsive">
-          <table id="dojoEventsTable" class="table cd-grid-table table-striped">
-            <thead>
-            <tr>
-              <th><span ng-click="toggleSort($event, 'name', dojoEvents.dojo.id)" ng-class="getSortClass(sort[dojoEvents.dojo.id]['name'])">&nbsp;</span>{@i18n key="Event"/}
-              </th>
-              <th><span ng-click="toggleSort($event, 'createdAt', dojoEvents.dojo.id)" ng-class="getSortClass(sort[dojoEvents.dojo.id]['createdAt'])">&nbsp;</span>{@i18n key="Date"/}
-              </th>
-              <th>{@i18n key="Type"/}</th>
-              <th></th>
-            </tr>
-            </thead>
-            <tbody ng-repeat="event in dojoEvents.events" ng-switch on="dojoEvents.events[$index].isCollapsed" class="events-info">
-            <tr ng-click="showEventInfo(dojoEvents, $index)" class="event-info-row">
-              <td>{{ event.name }}</td>
-              <td>{{ event.formattedDate }}</td>
-              <td>{{ eventTypeTranslations[event.type] }}</td>
-              <td class="pointer">
-                <i class="pull-right glyphicon ng-scope glyphicon-chevron-right" ng-class="{'glyphicon-chevron-down': events[$index].isCollapsed, 'glyphicon-chevron-right': !events[$index].isCollapsed}"></i>
-              </td>
-            </tr>
-            <tr ng-switch-when="true" class="event-info-row-content" ng-init="dojo = dojoEvents.dojo">
-              <td colspan="9">
-                <div class="span9">
-                  <cd-apply-for-event></cd-apply-for-event>
-                </div>
-              </td>
-            </tr>
-            </tbody>
-          </table>
-        </div>
+  <!--<div class="panel panel-body" ng-if="!dojosEvents || dojosEvents.length === 0 || !currentEvents">
```

Comment:
Comment was left in there in case you simply wanted the text moved, as opposed to a red alert.